test/sys/test_termios.rs

```diff
@@ -14,22 +15,18 @@ fn write_all(f: RawFd, buf: &[u8]) {
     }
 }
 
+// Test test_tcgetattr with multiple file descriptor types
```

Comment:
Since this describes multiple file descriptor types, can you enumerate each type within the test? I'd actually suggest breakaing this test into 3 different ones for each type instead of one big one.